app/src/main/java/com/infomaniak/drive/ui/fileList/FileAdapter.kt

```diff
@@ -280,13 +280,14 @@ open class FileAdapter(
                 setCorners(position, itemCount)
             }
 
+            fileChecked.isVisible = false
             setFileItem(file, isGrid, holder)
             checkIfEnableFile(file)
 
             when {
                 uploadInProgress && !file.isPendingUploadFolder() -> displayStopUploadButton(position, file)
                 multiSelectManager.isMultiSelectOn -> displayFileChecked(file, isGrid)
-                else -> displayFilePreview()
+                else -> if (!file.isImporting()) displayFilePreview()
```

Comment:
Won't it cause problems if we are in multiselection mode?

Reply:
it shouldn't because in multiselect mode, the else clause will not be reached